src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__name {
+      color: #34568b;
+    }
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $grey-color;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__review {
+    display: flex;
+    width: 166px;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    margin-top: 12px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
This value is default, so you can just remove it everywhere
```suggestion
```

Reply:
When I removed font-weight everything broke, so I returned it.